employee-rostering-server/src/main/resources/org/optaweb/employeerostering/server/solver/employeeRosteringScoreRules.drl

```diff
@@ -89,6 +91,64 @@ rule "No 2 shifts within 10 hours from each other"
         scoreHolder.addHardConstraintMatch(kcontext, -1);
 end
 
+rule "At most contract maximum minutes per day each day"
```

Comment:
Sounds strange. Maybe something like "Daily minutes must not exceed contract maximum"?

Reply:
That sounds better; will change